main.py

```diff
@@ -0,0 +1,19 @@
+from controller import *
```

Comment:
Typically, it's a good idea to only import what you need in a file, rather than using the asterisk wildcard (which means "import everything"). In this case, it's better to use:

```suggestion
from controller import Controller
```

You can fix this in your other files, too.